include/cantera/kinetics/Reaction.h

```diff
@@ -147,25 +147,25 @@ class Reaction
     std::string id;
 
     //! True if the current reaction is reversible. False otherwise
-    bool reversible;
+    bool reversible = true;
 
     //! True if the current reaction is marked as duplicate
-    bool duplicate;
+    bool duplicate = false;
 
     //! True if reaction orders can be specified for non-reactant species.
     //! Default is `false`.
-    bool allow_nonreactant_orders;
+    bool allow_nonreactant_orders = false;
 
     //! True if negative reaction orders are allowed. Default is `false`.
-    bool allow_negative_orders;
+    bool allow_negative_orders = false;
 
     //! Input data used for specific models
     AnyMap input;
 
     //! The units of the rate constant. These are determined by the units of the
     //! standard concentration of the reactant species' phases of the phase
     //! where the reaction occurs.
-    Units rate_units;
+    Units rate_units {0.0};
```

Comment:
I think the preferred style for the brace-initialization is to omit the space, analogous to the constructor initialization e.g.
```suggestion
    Units rate_units{0.0};
```